src/tests/test_piranha_python.rs

```diff
@@ -11,18 +11,47 @@ Copyright (c) 2022 Uber Technologies, Inc.
  limitations under the License.
 */
 
-use super::{create_match_tests, create_rewrite_tests, substitutions};
-
-use crate::models::default_configs::PYTHON;
-
-create_rewrite_tests!(
-  PYTHON,
-  test_delete_modify_str_literal_from_list:  "delete_cleanup_str_in_list", 1,
-  substitutions = substitutions! {
-    "str_literal" => "dependency2",
-    "str_to_replace"=>  "dependency1",
-    "str_replacement" => "dependency1_1"
-  };
-);
+use assert_cmd::prelude::{CommandCargoExt, OutputAssertExt};
+
+use std::{fs::File, path::Path, process::Command};
+use tempdir::TempDir;
+
+use super::create_match_tests;
+
+use crate::{
+  models::{default_configs::PYTHON, piranha_output::PiranhaOutputSummary},
+  utilities::{eq_without_whitespace, read_file},
+};
+
+#[test]
```

Comment:
An equivalent test case that uses CLI

Reply:
Equivalent to what? (Also, maybe that should be a comment in the source rather than on the PR?)